.github/workflows/test-coverage-enhancement.yml

```
@@ -51,10 +52,15 @@ jobs:
-          PR_NUMBER=$(gh pr view --json number -q '.number')
+          if [ "${{ github.event_name }}" == "pull_request" ]; then
+            PR_NUMBER=${{ github.event.pull_request.number }}
+          else
+            PR_NUMBER=$(gh pr view --json number -q '.number')
+          fi
           UNTESTED_COMPONENTS=$(cat untested-components.json)
           gh pr comment $PR_NUMBER --body "@coderabbitai generate tests for these untested files: \`\`\`json\n${UNTESTED_COMPONENTS}\n\`\`\`"
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.PAT_GITHUB }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)